Jenkinsfile

```diff
@@ -85,68 +81,53 @@ spec:
                 // development project id
                 env.PROJECT_ID = "${PROJECT_ID}"
                 env.REGION = "${REGION}"
-                def shortCommit = sh ( returnStdout: true, script: 'git rev-parse HEAD | cut -c 1-6').trim()
-                env.CLUSTER_NAME = "mycsharp-${shortCommit}"
-                //env.CLUSTER_NAME = "${CLUSTER_NAME}"
-                env.KEYFILE = "/home/jenkins/dev/jenkins-deploy-dev-infra.json"
+                env.KEYFILE = GOOGLE_APPLICATION_CREDENTIALS
             }
           // Setup gcloud service account access
           sh "gcloud auth activate-service-account --key-file=${env.KEYFILE}"
           sh "gcloud config set compute/zone ${env.CLUSTER_ZONE}"
           sh "gcloud config set core/project ${env.PROJECT_ID}"
-          sh "gcloud config set container/cluster ${env.CLUSTER_NAME}"
           sh "gcloud config set compute/region ${env.REGION}"
          }
         }
     }
 
-    stage('linter') {
+    stage('configure environment file') {
```

Comment:
All of this file needs code level docs

Reply:
Comments have been added to the configure environment script